### ISSUE TITLE
Latest version of addon isn't appearing in dashboard

### ISSUE THREAD

**Issue by ndarilek** [home-assistant/hassio#662]
1. Add https://github.com/bestlibre/hassio-addons.
2. Click Mopidy.

Expected result: Mopidy is shown as being version 0.2.4.

Actual result: It shows as being 0.1.2.

When I SSH into developer mode and look at the repository, I see the version set as 0.2.4. But for some reason, I'm only seeing 0.1.2.

I've uninstalled and reinstalled the repository several times now, and I don't have a version of Mopidy installed. Something seems to have cached this, and I don't know what.

Thanks.

**Comment by ndarilek** [comment]
FWIW, I reinstalled HassOS and this problem still persists. I literally just added https://github.com/bestlibre/hassio-addons/ as a repository immediately after first boot, clicked "Mopidy", and saw version 0.1.2 listed despite 0.2.4 being the latest in the repository. I also checked Docker Hub, and 0.2.4 is tagged along with _latest_.

There are also a few issues to this effect on the above repo as well, so it seems something fishy is going on with this particular addon.

My snapshot from before my switch to HassOS still has 0.2.4, so the addon should be installable. This is with 32-bit Hassio on an RPi 3B.

Thanks!

**Comment by pvizeli** [comment]
Press reload on add-on store and post the logs of supervisor. I think there will be a notice why you not see this version.

**Comment by ndarilek** [comment]
```
18-08-22 10:50:19 INFO (MainThread) [hassio.api.security] /addons/reload 
access from core_ssh
18-08-22 10:50:19 INFO (MainThread) [hassio.addons.git] Update addon 
https://github.com/bestlibre/hassio-addons repository
18-08-22 10:50:19 INFO (MainThread) [hassio.addons.git] Update addon 
https://github.com/hassio-addons/repository repository
18-08-22 10:50:19 INFO (MainThread) [hassio.addons.git] Update addon 
https://github.com/home-assistant/hassio-addons repository
18-08-22 10:50:19 INFO (MainThread) [hassio.addons.git] Update addon 
https://gitlab.com/ndarilek/hassio-mycroft repository
18-08-22 10:50:21 INFO (MainThread) [hassio.addons] Load addons: 64 all 
- 0 new - 0 remove
18-08-22 10:50:26 INFO (MainThread) [hassio.api.security] 
/supervisor/logs access from core_ssh
```

**Comment by tjorim** [comment]
> My snapshot from before my switch to HassOS still has 0.2.4, so the addon should be installable. This is with 32-bit Hassio on an RPi 3B.

Do you mean that before HassOS you installed the 32bit, or now as well?

**Comment by ndarilek** [comment]
I installed the 32-bit this time. I think there were only 32-bit 
installations with the Resin-based builds, but I can't say for sure.

**Comment by itanczos** [comment]
Indeed, the version number is badly indicated.
But it runs the new version.
If you start, LOG will show Mopidy 2.1.0.

The bigger problem is that it does not work! :(

**Comment by ndarilek** [comment]
I'm not convinced that, just because the log shows 2.1.0, you're getting 
version 0.2.4 of the addon. Version 0.1.2 of the addon very may well be 
2.1.0 of Mopidy since there's no version-pinning in the Dockerfile.

ANd, you're right, version 0.1.2 didn't work for me either, while 0.2.4 did.

**Comment by pvizeli** [comment]
Ah, the add-on version is 0.1.2 and inside this add-on run mopidy 0.2.4 ?

**Comment by ndarilek** [comment]
No, the addon version is 0.2.4 running Mopidy 2.1.0. I suspect 0.1.2 
runs 2.1.0 as well, though with a broken setup that doesn't actually work.

**Comment by tjorim** [comment]
Could it be related to https://github.com/bestlibre/hassio-addons/issues/80#issuecomment-416180889?

**Comment by frenck** [comment]
@ndarilek Have you opened an issue with the actual repository?
I'm running a lot of add-ons myself, and I'm unfamiliar with this issue (and I'm pretty sure I would have noticed).
